fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -228,7 +228,6 @@ __all__ = ["«cls.name»", "main"]
 
     def delete_device(self):
         """ Hook to delete resources allocated in init_device.
```

Comment:
Please remove leading space in docstring.  `"""Hook to ...```

Reply:
Which leading space you are referring to? Running flake8 is not giving any warning for any space.

Reply:
1. Leading space, after three quotes:  `""" Hook `
2. No leading space, after three quotes:  `"""Hook `

I prefer option 2.

Reply:
Done. Link to the commit
https://github.com/ska-telescope/pogo/commit/60b7ef2651fc88fc6d518d15e00c8fa216d384a4